hack/gen-api-reference-docs.sh

```diff
@@ -25,20 +25,20 @@ REFDOCS_REPO="https://${REFDOCS_PKG}.git"
 REFDOCS_VER="5c208a6"
 
 KNATIVE_SERVING_REPO="github.com/knative/serving"
-KNATIVE_SERVING_COMMIT="v0.2.3"
-KNATIVE_SERVING_OUT_FILE="serving.md"
+KNATIVE_SERVING_COMMIT="v0.3.0"
```

Comment:
ah I guess people will edit these numbers at some point :)

we can totally make this configurable with a default value, like 

```
KNATIVE_SERVING_COMMIT="${KNATIVE_SERVING_COMMIT:-v0.3.0}"
```

or make it required like:

```
KNATIVE_SERVING_COMMIT="${KNATIVE_SERVING_COMMIT?:specify KNATIVE_SERVING_COMMIT variable}"
```

either way is cool with me.

Reply:
I fixed this in my PR #998